vhdbuilder/packer/components.json

```diff
@@ -2,351 +2,425 @@
   "ContainerImages": [
     {
       "downloadURL": "mcr.microsoft.com/oss/kubernetes/kubernetes-dashboard:*",
-      "versions": [
+      "amd64OnlyVersions": [
         "v1.10.1"
-      ]
+      ],
+      "multiArchVersions": []
     },
     {
       "downloadURL": "mcr.microsoft.com/oss/kubernetes/dashboard:*",
-      "versions": [
+      "amd64OnlyVersions": [
         "v2.0.1"
+      ],
+      "multiArchVersions": [
+        "v2.4.0"
       ]
     },
     {
       "downloadURL": "mcr.microsoft.com/oss/kubernetes/metrics-scraper:*",
-      "versions": [
+      "amd64OnlyVersions": [
         "v1.0.4"
+      ],
+      "multiArchVersions": [
+        "v1.0.7"
       ]
     },
     {
       "downloadURL": "mcr.microsoft.com/oss/kubernetes/exechealthz:*",
-      "versions": [
+      "amd64OnlyVersions": [
         "1.2_v0.0.5"
-      ]
+      ],
+      "multiArchVersions": []
     },
     {
       "downloadURL": "mcr.microsoft.com/oss/kubernetes/autoscaler/addon-resizer:*",
-      "versions": [
+      "amd64OnlyVersions": [
         "1.8.5",
         "1.7"
+      ],
+      "multiArchVersions": [
+        "1.8.14"
       ]
     },
     {
       "downloadURL": "mcr.microsoft.com/oss/kubernetes/metrics-server:*",
-      "versions": [
-        "v0.3.6",
+      "amd64OnlyVersions": [
+        "v0.3.6"
+      ],
+      "multiArchVersions": [
         "v0.4.4"
       ]
     },
     {
       "downloadURL": "mcr.microsoft.com/k8s/core/pause:*",
-      "versions": [
+      "amd64OnlyVersions": [
         "1.2.0"
-      ]
+      ],
+      "multiArchVersions": []
     },
     {
       "downloadURL": "mcr.microsoft.com/oss/kubernetes/pause:*",
-      "versions": [
+      "amd64OnlyVersions": [
         "1.4.0",
         "3.5"
+      ],
+      "multiArchVersions": [
+        "3.6"
       ]
     },
     {
       "downloadURL": "mcr.microsoft.com/oss/kubernetes/coredns:*",
-      "versions": [
-        "1.6.6",
-        "1.8.3",
+      "amd64OnlyVersions": [
+        "1.6.6"
+      ],
+      "multiArchVersions": [
         "1.8.6"
       ]
     },
     {
       "downloadURL": "mcr.microsoft.com/containernetworking/azure-cns:*",
-      "versions": [
-        "v1.4.9",
```

Comment:
azure-cns was switch from 1.4.9 to 1.4.15 by azure cni team one month ago in addon chart in RP side. so delete it from VHD to save some vhd space. I removed some other container images for the same reason. 
{{- else if eq .component "azure-cns-image" -}}
v1.4.15